PictureApp/Controller/OptionsViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  OptionsViewController.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 26.11.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+protocol OptionsViewControllerDelegate : AnyObject {
+    func optionsViewController (_ controller: OptionsViewController, didSelectOrderStrategy order: Order)
+    func didFilterSelected()
+    
+}
+ 
+enum Order :String{
```

Comment:
Extra space after order and missing of space after `:`

Reply:
Done